### DISCUSSION TITLE
version problem

### DISCUSSION THREAD

**Discussion by obkduishinijf** [3kh0/ext-remover#363] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
using "v102" but it acts as v106+, anything i should do??

**Comment by Aka-but-nice** [comment, marked as answer]
Yea, ingot doesn't work on v106+ and v102.

**Reply by obkduishinijf** [reply]
well i've found a way, very tedious though.

**Reply by Alex2Mayhem** [reply]
> well i've found a way, very tedious though.

What way is that? tell us.

**Reply by obkduishinijf** [reply]
> > well i've found a way, very tedious though.
> 
> What way is that? tell us.

downgrading, took me too fucking long because i've got the intelligence of dirt

**Reply by Aka-but-nice** [reply]
Yall could get in trouble for that, try #347 or #348
